src/test/java/org/broadinstitute/hellbender/engine/filters/CountingReadFilterUnitTest.java

```diff
@@ -244,5 +247,62 @@ public void testSubFilterCounts(GATKRead[] reads, long totalRejections, long sta
         Assert.assertEquals(badStartAndEnd.getFilteredCount(), startEndRejections);
         Assert.assertEquals(isRayOrEgon.getFilteredCount(), nameRejections);
     }
+
+    @Test
+    public void testFromListNull() {
+        CountingReadFilter rf = CountingReadFilter.fromList(null, ArtificialReadUtils.createArtificialSamHeader(1, 1, 10));
+        Assert.assertTrue(rf.delegateFilter.getClass().getSimpleName().equals(
+                ReadFilterLibrary.AllowAllReadsReadFilter.class.getSimpleName()));
```

Comment:
You can assert equality on the class objects themselves using `==`, since they are singletons (here and below).